network/address_test.go

```diff
@@ -458,9 +478,19 @@ var selectInternalMachineTests = []selectTest{{
 		network.NewScopedAddress("fe80::1", network.ScopeLinkLocal),
 		network.NewScopedAddress("127.0.0.2", network.ScopeMachineLocal),
 	},
+	4,
+}, {
+	"first machine local non-IPv4 address is selected even with public/cloud hostnames",
```

Comment:
There are no IPv4 local-machine addresses here, so either the comment is wrong or something else?